packages/liquidation-sdk-viem/src/positions/getters.ts

```diff
@@ -63,21 +77,23 @@ async function getLiquidatablePositions(
           position.user.address,
           position.market.uniqueKey,
           client,
-          { chainId },
+          // Disable `deployless` so that viem multicall aggregates fetches
+          { chainId, deployless: false },
```

Comment:
In the backlog there is a ticket to make deployless reads plural based on your prior feedback, jsyk